src/Routing/Router.php

```diff
@@ -978,13 +1005,82 @@ public static function scope($path, $params = [], $callback = null)
         $options = [];
         if (is_array($params)) {
             $options = $params;
-            unset($params['routeClass'], $params['extensions']);
+            $cache = (string)Hash::get($params, '_cache');
+            unset($params['routeClass'], $params['extensions'], $params['_cache']);
+            if (!empty($cache) && static::_isCacheSerialize($cache)) {
```

Comment:
This would be simpler if the check for the cache is done in `rememberScope`
The end of the function would be this line, having something like `return static::rememberScope(...)`